Package.swift

```diff
@@ -29,10 +29,11 @@ let package = Package(
         )
     ],
     dependencies: [
-        .package(url: "https://github.com/IBM-Swift/Kitura.git", .upToNextMinor(from: "2.1.0")),
-        .package(url: "https://github.com/IBM-Swift/BlueCryptor.git", .upToNextMinor(from: "0.8.0")),
-        .package(url: "https://github.com/IBM-Swift/SwiftyJSON.git", .upToNextMajor(from: "17.0.0")),
```

Comment:
As far as I can tell, `.upToNextMajor(from: "17.0.0")` and `from: "17.0.0"` have the same meaning (https://github.com/apple/swift-package-manager/blob/master/Documentation/PackageDescriptionV4.md). The former is more descriptive (in my opinion), but the latter is more commonly used.

I'd be happy with either here, and I'd lean towards the more common expression so I'd support this line change.